Cargo.toml

```diff
@@ -1,7 +1,6 @@
 [package]
 name = "rustduino"
-version = "0.1.1"
-description="A generic HAL implementation for Arduino Boards in Rust"
```

Comment:
Why is the description begin removed and the version being yanked?

Reply:
Fixed